pysmt/smtlib/script.py

```diff
@@ -392,3 +419,36 @@ def evaluate_command(cmd, solver):
                                   "Please open a bug-report." % cmd.name)
     else:
         raise UnknownSmtLibCommandError(cmd.name)
+
+def omt_evaluate_command(cmd, optimizer, client_data = None):
```

Comment:
We discussed this: if you simply rename `client_data` into `optimization_goals` and make it non-optional you can simply avoid the ugly 
```
 if client_data is not None:
￼            client_data[0].append(rt)
```
And simply do `optimization_goals.append(rt)` when needed... Isn't this simpler?